pkg/server/plugin/bundlepublisher/awss3/awss3_test.go

```diff
@@ -98,6 +110,20 @@ func TestConfigure(t *testing.T) {
 			expectMsg:    "failed to create client: client creation error",
 			newClientErr: errors.New("client creation error"),
 		},
+		{
+			name: "wrong endpoint format",
```

Comment:
I find the format in the test name a bit confusing because there's a `Format` field in the config too. Maybe we can say:
```suggestion
			name: "invalid endpoint url",
```